src/renderer/atlas/AtlasEngine.r.cpp

```diff
@@ -162,6 +162,10 @@ void AtlasEngine::_recreateAdapter()
 
 void AtlasEngine::_recreateBackend()
 {
+    // D3D11 defers the destruction of objects and only one swap chain can be associated with a
+    // HWND, IWindow, or composition surface at a time. --> Destroy it while we still have the old device.
+    _destroySwapChain();

```

Comment:
We forgot to flush the device before recreating it (and the swap chain). It's also technically not allowed.